plugins/woocommerce/src/Internal/EmailEditor/Package.php

```diff
@@ -35,15 +33,16 @@ class Package {
 	 * @internal
 	 */
 	final public static function init() {
-		self::$package_active = FeaturesUtil::feature_is_enabled( 'block_email_editor' );
+		self::$package_active = get_option( 'woocommerce_feature_block_email_editor_enabled', 'no' ) === 'yes'; // init is called pretty early. Cant use FeaturesUtil.
```

Comment:
I assume we are switching to get_option because FeatureUtil triggers translations loading. Correct?

Reply:
Yes, this is correct.